cmd/kube-controller-manager/app/options/options.go

```diff
@@ -221,40 +223,41 @@ func NewDefaultComponentConfig() (kubectrlmgrconfig.KubeControllerManagerConfigu
 }
 
 // Flags returns flags for a specific KubeController by section name
-func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledByDefaultControllers []string) cliflag.NamedFlagSets {
+func (s *KubeControllerManagerOptions) Flags(allControllers []string, disabledByDefaultControllers []string, controllerAliases map[string]string) cliflag.NamedFlagSets {
 	fss := cliflag.NamedFlagSets{}
-	s.Generic.AddFlags(&fss, allControllers, disabledByDefaultControllers)
+	s.Generic.AddFlags(&fss, allControllers, disabledByDefaultControllers, controllerAliases)
 	s.KubeCloudShared.AddFlags(fss.FlagSet("generic"))
-	s.ServiceController.AddFlags(fss.FlagSet("service controller"))
+	s.ServiceController.AddFlags(fss.FlagSet(cpnames.ServiceControllerName))
```

Comment:
this will make the CLI `--help` sections consistent with the controller names: 

```
Pod-garbage-collector-controller flags:

      --terminated-pod-gc-threshold int32   Number of terminated pods that can exist before the terminated pod garbage collector starts deleting terminated pods. If <= 0, the terminated pod garbage collector is disabled. (default 12500)
```

One disadvantage is that it has the dash - I could make a function that would split this but we would lose `ctrl-f` functionality for controllers